.github/workflows/commit-validation.yml

```diff
@@ -8,8 +8,6 @@ on:
   pull_request:
     branches:
       - "**"
-    paths-ignore:
-      - "docs/**"
```

Comment:
this is removed in this PR so CI runs on this change which is docs only. 

https://github.com/typeorm/typeorm/pull/11761 will update the workflow so it correctly decides which jobs to run on docs only changes